### PR TITLE
feat(breadcrumbs): add optional vertical gap

### PR DESCRIPTION
**Pull request by fulopdaniel**

### CODE REVIEW

**Review by celine-s — APPROVED** [review]
lgtm. 
Should someone from the fondue team also have a look.?

File: src/components/Breadcrumbs/Breadcrumbs.cy.tsx [hunk 1]
```diff
@@ -95,6 +96,16 @@ describe('Breadcrumb component', () => {
         cy.get(BREADCRUMB_ITEM_ID).eq(2).find('a').should('exist');
     });
 
+    it('should render gap class appropriately', () => {
+        for (const gap of Object.entries(BreadcrumbGap)) {
```
Comment (imoutaharik) [inline src/components/Breadcrumbs/Breadcrumbs.cy.tsx:100]: Using for loop in cypress makes the tests flaky, we try to avoid it.

**Review by imoutaharik — CHANGES_REQUESTED** [review]
Thanks for adding the option! 🎉 
I just made a comment for the tests.

**Review by imoutaharik — APPROVED** [review]
Thanks!

**Pull request merged**